docs/articles/documentation/test-api/assertions/index.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: docs
+title: Assertions
+permalink: /documentation/test-api/assertions/
+checked: false
+---
+# Assertions
+
+To check if the state of the tested webpage matches the one you expect to see, use *assertions*.
+
+TestCafe provides a comprehensive [set of assertions](assertion-api.md) and all of them based on Behavior Driven Development style (BDD-style).
+
+This topic contains the following sections.
+
+* [Assertion timeout](#assertion-timeout)
+* [Assertion options](#assertion-options)
+  * [options.timeout](#optionstimeout)
+
```

Comment:
It seems strange to me that we write 
`This topic contains the following sections.`
... and there are no basic section.

Maybe it's better to do like in [Selectors article](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html)?

```md
* [Creating Assertions](#creating-assertions)
* [Assertion timeout](#assertion-timeout)
* [Assertion options](#assertion-options)
  * [options.timeout](#optionstimeout)
```